### PR TITLE
fix(dashboard): remove fixed button width, revers manage button order

### PR DESCRIPTION
**Pull request by MauserBitfly**
This PR:
- adapts the validator dashboard header
  - removes the fixed button width for buttons
  - reverses the sort of the manage buttons on mobile screen (when they are in a dropdown)

BEDS-285

### CODE REVIEW

File: frontend/components/dashboard/DashboardControls.vue [hunk 1]
```diff
@@ -90,7 +90,7 @@ const manageButtons = computed<MenuBarEntry[] | undefined>(() => {
     return [ {
       dropdown: true,
       highlight: true,
-      items: buttons,
+      items: buttons.reverse(), // In the dropdown we want the button sorting to be reversed
```
Comment (marcel-bitfly) [inline frontend/components/dashboard/DashboardControls.vue:93]: nitpick: creating layout or changing layout in `script`-part is confusing. 
The layout should be defined through composition of `components`.

**Review by marcel-bitfly — APPROVED** [review]
👍

**Comment by marcel-bitfly** [comment]
nitpick: typo in `commit msg`

File: frontend/components/dashboard/DashboardControls.vue (hunk 1 above)
Comment (MauserBitfly) [reply frontend/components/dashboard/DashboardControls.vue:93]: for simplicity I leave it here. ;-)

**Pull request merged**